src/components/Footer/index.tsx

```diff
@@ -0,0 +1,114 @@
+"use client"
+import Image from "next/image";
+import Link from "next/link";
+import Logo from '../../../public/images/parking-time-logo.png'
+import { FiInstagram } from "react-icons/fi";
+import { FaLinkedin } from "react-icons/fa";
+import { useTranslations, useLocale } from "next-intl";
+import { usePathname } from "next/navigation";
+
+type NavItemDesk = {
+    name: string;
+    link: string;
+  };
+  
+  const navItem: NavItemDesk[] = [
+    // {
+    //   name: 'Contact',
```

Comment:
Is this going to be used later? Otherwise just remove it :)

Reply:
Its going to be used later when contact page is up :)